.github/workflows/firebase-prod-merge.yml

```diff
@@ -32,7 +33,10 @@ jobs:
           cp .npmrc functions/.npmrc
           echo "//npm.pkg.github.com/:_authToken=${{ secrets.GH_TOKEN_DISCO_MIR }}" >> functions/.npmrc
       - name: Build
+        # Make sure content is tracking main and up-to date, then builds and deploys 
         run: |
+          git config -f .gitmodules submodule.content.branch main
```

Comment:
would [this](https://git-scm.com/docs/git-submodule#Documentation/git-submodule.txt-set-branch-b--branchltbranchgt--ltpathgt) be more idiomatic?

Reply:
That indeed is nicer. I was looking for a simpler command, I don't know how I missed it!. Thanks!